ui/raidboss/raidboss_config.js

```diff
@@ -554,7 +554,7 @@ class RaidbossConfigurator {
           div.appendChild(input);
           input.type = 'text';
           input.step = 'any';
-          input.placeholder = this.base.translate(kMiscTranslations.valueDefault);
+          input.placeholder = `${trig.durationSeconds || 3}${this.base.translate(kMiscTranslations.valueDefault)}`;
```

Comment:
There's a couple of reasons that I said default (mostly because it's easier and I was being lazy <_<).
* `durationSeconds` can be a function (that could depend on `data` and `matches`), and I'm not sure what you output for that
* default trigger duration can be set differently for alarm/alert/info, and so it's not necessarily 3 and may depend on what the trigger outputs.  (What if it outputs both info and alert and you have set those to different duration lengths?)

You could say:

if the trigger has no `durationSeconds` property and all of the alarm/alert/info options are set to the same value, then use that value.  Otherwise, if `durationSeconds` is a number, display that number.  Otherwise, display default.  This would cover many cases, but still avoid the trickier ones.

Reply:
Oh sorry I don't know default duration can be changed.
I think only show number for non default duration will be fine?

Reply:
Yeah, I think showing `trigger.durationSeconds` if `typeof trigger.durationSeconds === 'number'` sounds good to me.  Is that what you're saying?

Reply:
Seems checking with `typeof` is better here. Done.